procedures/foreman/branch.md.erb

```diff
@@ -39,14 +41,15 @@
 * [ ] [Generate](https://github.com/theforeman/theforeman-rel-eng/blob/master/generate_gpg), [backup](https://github.com/theforeman/theforeman-rel-eng/blob/master/export_gpg_private), [sign](https://github.com/theforeman/theforeman-rel-eng/blob/master/sign_gpg) and [upload](https://github.com/theforeman/theforeman-rel-eng/blob/master/upload_gpg) a GPG key. See [GPG_Keys](https://projects.theforeman.org/projects/foreman/wiki/GPG_Keys) if needed.
 * [ ] Publish the key by [exporting](https://github.com/theforeman/theforeman-rel-eng/blob/master/export_gpg_public) the GPG key
   * [ ] Update the website's [security.md](https://github.com/theforeman/theforeman.org/blob/gh-pages/security.md) and create a file in [static/keys](https://github.com/theforeman/theforeman.org/tree/gh-pages/static/keys)
-  * [ ] Create [releases/<%= release %>/RPM-GPG-KEY-foreman](https://yum.theforeman.org/releases/<%= release %>/RPM-GPG-KEY-foreman) on yum.theforeman.org
```

Comment:
Nit: there is a slash missing in front of var and the whole path could go in `backticks` for better recognition.